aws/templates/container/container_jenkins.ftl

```diff
@@ -0,0 +1,72 @@
+[#case "jenkins"]
+    [@Attributes image="jenkins-master" /]
+   
+    [#assign credentials = credentialsObject]
```

Comment:
This will break when we merge the settings stuff. I've left the credentialsObject in place but ideally it should be picking stuff up from the settings. Also the credentials will automatically be in the context so no need to use @Variables